webui/react/src/components/DeterminedAuth.tsx

```diff
@@ -51,7 +51,7 @@ const DeterminedAuth: React.FC<Props> = ({ canceler }: Props) => {
           type: StoreAction.SetAuth,
           value: { isAuthenticated: true, token, user },
         });
-        fetchMyRoles();
+        await fetchMyRoles();
```

Comment:
I put an `if (rbacEnabled) { ... }` around this in #5291  so just fyi before merging

Reply:
Added, thanks!